recipes/backports.unittest_mock/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.3" %}
+{% set sha256 = "c6d964d75112051576daa60746cc6a66afcdc43f13ea2bda175ca8e278e9b375" %}
+
+package:
+  name: backports.unittest_mock
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/b/backports.unittest_mock/backports.unittest_mock-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm >=1.15.0
+  run:
+    - python
+    - mock  # [py27]
+
+test:
+  imports:
+    - backports.unittest_mock
+
+about:
+  home: https://github.com/jaraco/backports.unittest_mock
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
With the `1.4` release tarball, we can make this simply `license_file: LICENSE`.